pkg/cluster/internal/create/nodes.go

```diff
@@ -209,7 +214,16 @@ func nodesToCreate(cfg *config.Config, clusterName string) []nodeSpec {
 		})
 	}
 
-	// TODO(bentheelder): handle implicit nodes as well
+	// add an external load balancer if there are multiple control planes
+	if controlPlanes > 1 {
```

Comment:
There should be a note, probably on the Role field or on the Nodes field, to explain that if you specify more than 1 control plane node then a load balancer will be provisioned.

IMO, API docs should themselves serve users quite well as a reference for how things work (even if we also have additional proper documentation for this too).

Reply:
done. the API docs definitely need work